spec/controllers/state_file/questions/nj_household_rent_own_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjHouseholdRentOwnController do
+  let(:intake) { create :state_file_nj_intake }
+  before do
+    sign_in intake
+  end
+
+  describe "#edit" do
+    render_views
+    it 'succeeds' do
+      get :edit
+      expect(response).to be_successful
```

Comment:
[pebble] is this test sufficient? other controller tests (looking at county and municipality) are also testing that the update function is in fact updating the correct intake params